GVRf/Framework/framework/build.gradle

```diff
@@ -73,7 +76,7 @@ android {
     }
 
     sourceSets.main {
-        java.srcDirs = ['src/main/java', 'src/main/backends/oculus']
+        java.srcDirs = ['src/main/java', '../' + rootProject.property("GVRF_BACKEND")+'/src/main/java']

```

Comment:
- Yes, backend 2 needs the same folder structure
- Dynamic loading and switching will be in the future; for now we will probably bundle Google VR into the Oculus backend; like monoscopic. Eventually both monoscopic and Google VR will be separate.
- No